<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>⏮</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>⏯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopPushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>⏹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="forwardButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>⏭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openPushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>⏫</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="timeSeriesPlot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>451</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="frameFreqPlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>451</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>170</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="slider_00032">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_00064">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_00128">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_00256">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_00512">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_01024">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_02048">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_04096">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_08192">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider_16384">
       <property name="minimum">
        <number>-16</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="PlotWidget" name="filterPlot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>451</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>150</y>
      <width>411</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>32</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>64</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>128</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>256</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>512</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>1024</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>2048</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>4096</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>8192</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>16384</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_00032">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>330</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_00064">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>330</y>
      <width>33</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_00128">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_01024">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>330</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_00512">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>330</y>
      <width>33</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_00256">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>330</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_08192">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>330</y>
      <width>33</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_04096">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>330</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_02048">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>330</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16384">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>330</y>
      <width>34</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_high_db">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>16dB</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_low_db">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>330</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
